File: CoreJava/game-tic-tac-toe/src/com/techlab/game/Mark.java
package com.techlab.game;

public enum Mark {

	NOUGHT, CROSS, BLANK;

	
	

	@Override
	public String toString() {
		switch (this) {
		case NOUGHT :
			return "O";
		case CROSS:
			return "X";
		case BLANK:
			return "-";
		default:
			return "Unspecified";
		}

	}

}

File: C#Folder/TicTacToeGameSolution/TicTacToeGameLib/Mark.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TicTacToeGameLib
{
    public enum Mark
    {
        NOUGHT, CROSS, BLANK
    }

    public static class MarkVal
    {
        public static string putString(this Mark m)
        {
            switch (m)
            {
                case Mark.NOUGHT:
                    return "O";
                case Mark.CROSS:
                    return "X";
                case Mark.BLANK:
                    return "-";
                default:
                    return "invalid";
            }
        }
    }

}
